content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -114,7 +114,7 @@ webhooks:
     service:
       namespace: "example-namespace"
       name: "example-service"
-    caBundle: "Ci0tLS0tQk...<base64-encoded PEM bundle containing the CA that signed the webhook's serving certificate>...tLS0K"
+    caBundle: "Ci0tLS0tQk...<`caBundle` is a PEM encoded CA bundle which will be used to validate the webhook's server certificate.>...tLS0K"
```

Comment:
YAML supports comments, so you could write:
```suggestion
    # caBundle is a PEM encoded certificate authority bundle
    # that is used to validate the webhook's server certificate.
    caBundle: "egCi0tLS0tQk...<more data>...tLS0K"
```

How does that look?

Reply:
LGTM. I wanted to leave the "formatting" as before.
Anyhow, this PR was already approved. I'll keep it for next time :)